src/RestEaseClientGenerator/Mappers/ModelsMapper.cs

```diff
@@ -30,6 +30,21 @@ internal class ModelsMapper : BaseMapper
     {
         foreach (var entry in schemas.OrderBy(s => s.Key))
         {
+            if (entry.Key == "ArrayOfCombinedTrips")
```

Comment:
I don't quite get these changes, are you defining domain specific logic in the generic mapper? Shouldn't the type of schema be handled in a correct way instead of handling this as an edge case?

Reply:
Sorry, this was just some left-over debug code, which can be removed in final version.